br/pkg/lightning/importer/import.go

```diff
@@ -2125,10 +2125,11 @@ func (rc *Controller) DataCheck(ctx context.Context) error {
 	if err := rc.checkTableEmpty(ctx); err != nil {
 		return common.ErrCheckTableEmpty.Wrap(err).GenWithStackByArgs()
 	}
-	if err := rc.checkCSVHeader(ctx); err != nil {
-		return common.ErrCheckCSVHeader.Wrap(err).GenWithStackByArgs()
+	if !rc.cfg.Mydumper.CSV.Base64Encoded {
+		if err := rc.checkCSVHeader(ctx); err != nil {
```

Comment:
Hi, as your definition

> This means all columns in csv files are base64 encoded. So lightning will decode it before storing it in the database.

we should also support the header is base64 encoded?